tests/dotnet-e2e/fileops/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿var TEXT = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.";
+
+// Read
+
+{
+    var dat = File.ReadAllText("/app/test.txt");
+
+    if (dat != TEXT)
+        throw new Exception($"Expected {TEXT}, got {dat}");
+}

```

Comment:
Not completly sure how dotnet works, but is there a testing framework? why not have proper function names?

Reply:
There are three primary testing frameworks in the dotnet ecosystem (atm).

- NUnit
- xUnit
- MsTest

There are pros and cons to using each of them. None of them are included out-of-the-box in a default console application template like I've constructed here. Adding any of them would involve more dependencies, increased project complexity, and running different commands against the project root- not in a "build this file and execute it" format like is currently the arrangement with the Go tests.

All that said, using one of these frameworks would add complexity while ultimately doing the same thing I've done here with vanilla operators and Exception throwing.

---

As for functions vs scopes, that was just a style choice 😄  I will get that changed 👍🏻

Reply:
sounds good, I guess we dont need a framework then